<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BRIMACLAP042.BRILLIO.COM" failures="0" tests="2" name="howToDoInJava.testNg.TimeoutMethod" time="0.932" errors="1" timestamp="23 Jul 2019 04:47:12 GMT" skipped="0">
  <testcase classname="howToDoInJava.testNg.TimeoutMethod" name="timeTestTwo" time="0.412"/>
  <testcase classname="howToDoInJava.testNg.TimeoutMethod" name="timeTestOne" time="0.520">
    <error message="Method howToDoInJava.testNg.TimeoutMethod.timeTestOne() didn&#039;t finish within the time-out 500" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method howToDoInJava.testNg.TimeoutMethod.timeTestOne() didn't finish within the time-out 500
at java.base@10.0.1/java.lang.Thread.interrupt(Thread.java:1016)
at app//org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:64)
at app//org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.base@10.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:514)
at java.base@10.0.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@10.0.1/java.lang.Thread.run(Thread.java:844)
]]>
    </error>
  </testcase> <!-- timeTestOne -->
</testsuite> <!-- howToDoInJava.testNg.TimeoutMethod -->
